paasta_tools/utils.py

```diff
@@ -2854,13 +2854,33 @@ def to_bytes(obj: Any) -> bytes:
         return str(obj).encode('UTF-8')
 
 
+TLS = threading.local()
```

Comment:
howcome we need this to be thread local?

Reply:
If `paasta_print()` is called from different threads, then it could have different `paasta_print_file` object sets by context managers in different threads. So, `paasta_print_file` needs to be a per-thread variable.

Reply:
I.e. let imagine that `paasta_print()` is simultaneously called from two threads serving two `/metastatus` HTTP API requests from two clients.  Is this case, we have two separate `io.StringIO` objects containing responses to corresponding clients, one in every thread, and we need `print_paasta()` to write it's output into a corresponding `io.String` output object.